openedx/core/djangoapps/content_libraries/models.py

```diff
@@ -527,3 +528,30 @@ def update_score(self, weighted_earned, weighted_possible, timestamp):
 
     def __str__(self):
         return str(self.usage_key)
+
+
+class ContentLibraryLearningPackage(models.Model):
```

Comment:
@ormsbee With Blockstore, a published library version can be identified by an int, corresponding to BundleVersion.version. That's what gets stored in LibraryContentBlock.source_library_version.

With this PR (which I know is very much a WIP), it looks like each *component* would have a version_num available (through PublishableEntity), but the library itself wouldn't--we'd need to look at the corresponding PublishLog.uuid for "published" versions and store that UUID in in LibraryContentBlock.source_library_version.

Does that sound right? Alternatively, would we want ContentLibraries themeslves (and courses, for that matter) to be PublishableEntities, making version_num available?